composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "sallar/laravel-jdate",
+    "type": "library",
+    "description": "This package helps developers to easily work with Jalali (Shamsi or Iranian) dates in Laravel applications",
+    "keywords": [
+        "sallar",
+        "laravel-jdate"
+    ],
+    "homepage": "https://github.com/sallar/laravel-jdate",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "sallar",
+            "email": "findme@sallar.me",
```

Comment:
my correct email is `sallar.kaboli@gmail.com` and maybe add yourself also as a contributor.